packages/@aws-cdk/cdk/lib/private/resolve.ts

```diff
@@ -36,12 +36,18 @@ export function resolve(obj: any, options: IResolveOptions): any {
   /**
    * Make a new resolution context
    */
-  function makeContext(appendPath?: string): IResolveContext {
+  function makeContext(appendPath?: string): [IResolveContext, IPostProcessor] {
```

Comment:
Can we avoid returning a tuple, and instead return some interface? The proliferation of `[0]` pains me...

Reply:
It's private though.

Reply:
I still have to read all those `[0]` :)